core/src/main/java/io/undertow/websockets/core/protocol/version07/WebSocket07FrameSinkChannel.java

```diff
@@ -41,7 +41,7 @@
     private final Masker masker;
     private volatile boolean dataWritten = false;
     protected final ExtensionFunction extensionFunction;
-    private final Random random = new Random();
+    private static final Random random = new Random();
```

Comment:
Would we be better off replacing the one use of this random instance with `ThreadLocalRandom.current().nextInt()`?

Reply:
From [Random javadoc](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html):

> Instances of java.util.Random are threadsafe. However, the concurrent use of the same java.util.Random instance across threads may encounter contention and consequent poor performance. Consider instead using ThreadLocalRandom in multithreaded designs.

Reply:
y, that's faster, let me update the pr.